### ISSUE TITLE
WordPress optimisation and caching (Feature Request)

### ISSUE THREAD

**Issue by lwj5** [bitnami/bitnami-docker-wordpress#262]
FEATURE REQUEST
**Description**
TLDR. 
- Consider enabling `mod_expires` and `mod_ext_filter` by default. 
- Consider installing brotli for better compression

If you see Bitnami's post on optimizing WordPress [here](https://docs.bitnami.com/aws/apps/wordpress/troubleshooting/optimize-bitnami-wordpress/), you will notice that the post recommends using WP caching plugins and .htaccess additions. To take advantage of these caching plugins and .htaccess rules for WordPress optimization, both `mod_expires` and `mod_ext_filter` must be enabled. However, these critical components do not come enabled by default and might give users a false impression that their WordPress sites are indeed optimized.

A power user might be able to see through that by using lighthouse or gtmetrix reports. Without these modules enabled the site will rank F in **Leverage browser caching**. Please consider turning these mods on by default.

Secondly, there are advantages in leveraging newer compression tech like Brotli. See
- https://blogs.akamai.com/2016/02/understanding-brotlis-potential.html
- https://blog.cloudflare.com/results-experimenting-brotli/

It is good to have but not essential, but I hope you will consider having that in the container as well.

**Comment by marcosbc** [comment]
Thanks for the suggestions! For now, you can enable the modules by following these steps: https://github.com/bitnami/bitnami-docker-wordpress/issues/130

As for Brotli, we'll evaluate whether it makes sense to include it in Bitnami images by default.
